tools/onnx-subgraph/include/device.h

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef DEVICE_H
+#define DEVICE_H
+
+#include <vector>
+#include <string>
+#include <iostream>
+#include <fstream>
+#include "onnx.pb.h"
+#include "graph.h"
+#include "json.h"
+enum class DeviceType
```

Comment:
```suggestion
#include "json.h"

enum class DeviceType
```

please add some empty lines between codes, like here and between methods/functions 
so that reading is better.
too tight codes are hard to read.
plz apply to all other codes.